oceanbasecloud/README.md

```diff
@@ -0,0 +1,42 @@
+# OceanBase Cloud Integration
+
+## Overview
+
+[OceanBase Cloud][1] provides fully managed database services with elastic scalability, ultra-fast performance, and high compatibility on global cloud infrastructure.
+
+Use the OceanBase Cloud Datadog integration to export metrics from OceanBase Cloud clusters to Datadog.
+
+## Setup
+
+To set up the OceanBase Cloud Datadog integration for your cluster, provide a Datadog API key and region to OceanBase Cloud.
+
+Go to [OceanBase Cloud Integrations][2] to configure the Datadog integration for your OceanBase Cloud project.
```

Comment:
Does this link lead the user directly to where they would add their API key and site information? If not, could you outline the steps a user should take after following this link?

Reply:
users do not need to provide their api keys, we have implemented our oauth client, README has been updated